src/app/pages/Home/Home.tsx

```diff
@@ -11,11 +11,30 @@ import Navigation from '../../components/Navigation/Navigation';
 import useCodeCard from '../../hooks/useAddCard';
 import type { CodeCards } from '../../types';
 import AddButton from '../../components/Buttons/AddButton/AddButton';
+import CodeField from '../../components/CodeField/CodeField';
+import SpinnerIcon from '../../components/assets/Spinner';
 
 export default function Home(): JSX.Element {
-  const { codeCards, removeCodeCard } = useCodeCard();
+  //Modal Toggle useStates
+  const [deleteModalToggle, setDeleteModalToggle] = useState(false);
+  const [editModalToggle, setEditModalToggle] = useState(false);
+  const [showCollectionModalToggle, setShowCollectioModalToggle] =
+    useState(false);
```

Comment:
I like the approach in the third state, with "show" in the name. `showDeleteModal` for example sounds much better to me